tutorials/dataframe/distrdf003_dask_lxbatch.py

```diff
@@ -0,0 +1,103 @@
+##   (https://jobqueue.dask.org).
+## - Access to the CERN batch service and specific configuration details,
+##   described at https://batchdocs.web.cern.ch/specialpayload/dask.html.
+##
+## The tutorial defines resources that each job will request to the condor
+## scheduler, then creates a Dask client that can be used by RDataFrame to
+## distribute computations.
+##
+## \macro_code
+## \macro_image
+##
+## \date March 2022
+## \author Vincenzo Eduardo Padulano
+import socket
+
+from dask.distributed import Client
+from dask_jobqueue import HTCondorCluster
+
+import ROOT
+RDataFrame = ROOT.RDF.Experimental.Distributed.Dask.RDataFrame
+
+
+def create_connection():
+    """
+    Creates a connection to HTCondor cluster offered by the CERN batch service.
+    Returns a Dask client that RDataFrame will use to distribute computations.
+    """
+    # The resources described in the specified arguments to this class represent
+    # the submission of a single job and will spawn a single Dask worker when
+    # the condor scheduler launches the job. Specifically, this example has Dask
+    # workers each with 2 cores (running in multiprocessing) and 2 GB of memory.
+    cluster = HTCondorCluster(cores=2, processes=2, memory="2GB", disk="0.1GB",
+                              job_extra={
+                                  "+JobFlavour": '"espresso"',
+                                  'should_transfer_files': 'Yes',
+                                  'when_to_transfer_output': 'ON_EXIT',
+                                  'getenv': 'True'
+                              },
+                              death_timeout='60',
+                              scheduler_options={
+                                  'port': 8786,
+                                  'host': socket.gethostname()
+                              },
+                              extra=['--worker-port 10000:10100']
+                              )
+
+    # The scale method allows to launch N jobs with the description above (thus
+    # N Dask workers). Calling this method on the cluster object launches the
+    # condor jobs (i.e. it is equivalent to `condor_submit myjob.sub`). In this
+    # example, two jobs are requested so two Dask workers will be eventually
+    # launched for a total of 4 cores.
+    cluster.scale(jobs=2)
+
+    # The Dask client can be created after the condor jobs have been submitted.
+    # At this point, the jobs may or may not have actually started. Thus, it is
+    # not guaranteed that the application already has the requested resources
+    # available.
+    client = Client(cluster)
+    # It is possible to tell the Dask client to wait until the condor scheduler
+    # has started the requested jobs and launched the Dask workers.
+    # The client will wait until 'n_workers' workers have been launched. In this
+    # example, the client waits for 1 worker before continuing with the
+    # application.
+    client.wait_for_workers(n_workers=1)
```

Comment:
should we teach this? if you want to mention it, should the code be commented to clarify you don't usually want to run it?